addons/web_tour/static/src/tour_service/tour_helpers.js

```diff
@@ -137,7 +137,10 @@ export class TourHelpers {
         });
         await moveTo(target, options);
         await dragEffectDelay();
-        await drop(target, options);
+        const dragHelperEl = document.body.querySelector(
+            ".o_draggable_dragging .o_snippet_thumbnail"
+        );
+        await drop(dragHelperEl, options);
```

Comment:
Do you still need this change ? I did a PR to remove the warning in this type of situation